sonar-docs/analysis/scan/sonarscanner-for-azure-devops.md

```diff
@@ -104,29 +104,42 @@ Once all this is done, you can trigger a build.
 
 Once all this is done, you can trigger a build.
 
-## Analyzing a Visual C++ project
+## Analyzing a C++ project
+In your build pipeline, insert the following steps in the order they appear here. These steps can be interleaved with other steps of your build as long as the following order is followed.
 1. Make **Build Wrapper** available on the build agent
-   * Download and unzip **Build Wrapper** on the build agent (see *Prerequisites* section of *C/C++/Objective-C* page). For the Microsoft-hosted build agent you will need to do it every time (as part of build pipeline), e.g. you can add **PowerShell script** task doing that. For the self-hosted build agent you can do the same either every build or only once (as part of manual setup of build agent). Example of PowerShell commands:
+   Download and unzip **Build Wrapper** on the build agent (see *Prerequisites* section of *C/C++/Objective-C* page). This can be done by inserting a **Command Line** task.\
```

Comment:
Why are you mentioning the command line task now, even if it might not be necessary? I would put it after the bullet points about self hosted vs MS-hosted